### DISCUSSION TITLE
import tg, unit ID spec

### DISCUSSION THREAD

**Discussion by 8555726** [chuot/rdio-scanner#284] [General]
Hi guys, does anyone have the import specification to enable importing of existing data. Tried a few options however it creates a new system and imports the data to the new system not the existing one.

For the unit ID's you can select the system and they import OK

**Comment by chuot** [comment]
Unfortunately it can be done with Rdio Scanner v6. I will make sure to add a system select box to the talkgroups import section for the next v7 release, which I'm working on. For now, you can delete the existing system and reimport all the talkgroup to a new system, which you will set to the same system ID as the older one. It should work, but make a backup first to be sure.

**Reply by 8555726** [reply]
ok, great thanks mate, purchased the iPhone app works a treat :)
